src/shared/hcm-show-api-service.ts

```diff
@@ -0,0 +1,155 @@
+import { SyncedRecordsResponse } from '@/shared/api-service-base';
+import { FlatfileEvent } from '@flatfile/listener';
+import axios from 'axios';
+import invariant from 'ts-invariant';
+
+// type DepartmentResult = {
+//   externalDepartmentId: string;
+//   name: string;
+//   description: string;
+// };
+
+// type EmployeeResult = {
+//   flatfileRecordId: string;
+//   firstName: string;
+//   lastName: string;
+//   phoneNumber: string;
+//   departmentId: string;
+//   jobId: string;
+// };
+
+// type JobResult = {
+//   externalJobId: string;
+//   name: string;
+//   description: string;
+// };
```

Comment:
Typing the API responses with a `as Jobresult etc` can help in the functions they're called

Reply:
Ah thanks, meant to add them back in